packages/modules/core/src/components/App.js

```diff
@@ -86,15 +98,53 @@ export const DigitApp = ({ stateCode, modules, appTenants, logoUrl }) => {
         </div>
       </Route>
       <Route path="/digit-ui/citizen">
-        <TopBar
-          img={cityDetails?.logoId}
-          ulb={`${t(cityDetails?.i18nKey)} ${ulbCamel(t("ULBGRADE_MUNICIPAL_CORPORATION"))}`}
-          isMobile={true}
-          toggleSidebar={() => toggleSidebar(!isSidebarOpen)}
-          logoUrl={logoUrl}
-          onLogout={handleLogout}
-          userDetails={userDetails}
-        />
+        {mobileView && (
+          <TopBar
+            img={cityDetails?.logoId}
+            ulb={`${t(cityDetails?.i18nKey)} ${ulbCamel(t("ULBGRADE_MUNICIPAL_CORPORATION"))}`}
+            isMobile={true}
+            toggleSidebar={() => toggleSidebar(!isSidebarOpen)}
+            logoUrl={logoUrl}
+            onLogout={handleLogout}
+            userDetails={userDetails}
+          />
+        )}
+        {!mobileView && (
+          <div className="topbar">
+            <img className="city" src={cityDetails?.logoId} />
+            <span className="ulb">
+              {t(cityDetails?.i18nKey)} {ulbCamel(t("ULBGRADE_MUNICIPAL_CORPORATION"))}
+            </span>
+            <div className="right width-20 flex-right">
+              <div className="left">
+                <ChangeLanguage dropdown={true} />
+              </div>
+              <div className="left width-20">
+                <Dropdown
```

Comment:
We should use the new component you made catered to this use case, rather than freeze option

Reply:
I haven't created a new component just added a new option in the props